utils/default values/songs.js

```diff
@@ -1,90 +1,48 @@
+  "xStep",
+  "Clutterfunk",
+  "Theory of Everything",
+  "Electroman Adventures",
+  "Clubstep",
+  "Electrodynamix",
+  "Hexagon Force",
+  "Blast Processing",
+  "Theory of Everything 2",
+  "Geometrical Dominator",
+  "Deadlocked",
+  "Fingerdash",
+  "Explorers",
+  "Firebird",
+  "The Seven Seas",
+  "Viking Arena",
+  "Airborne Robots",
+  "Secret",
+  "Payload",
+  "Beast Mode",
+  "Machina",
+  "Years",
+  "Frontlines",
+  "Space Pirates",
+  "Striker",
+  "Embers",
+  "Round 1",
+  "Monster Dance Off",
+  "Press Start",
+  "Nock Em",
+  "Power Trip",
+  "Stereo Madness"
+]
 function init(songID) {
-    switch (songID) {
-        case 0:
-            return "Stay Inside Me";
-        case 1:
-            return "Stereo Madness";
-        case 2:
-            return "Back On Track";
-        case 3:
-            return "Polargeist";
-        case 4:
-            return "Dry Out";
-        case 5:
-            return "Base After Base";
-        case 6:
-            return "Cant Let Go";
-        case 7:
-            return "Jumper";
-        case 8:
-            return "Time Machine";
-        case 9:
-            return "Cycles";
-        case 10:
-            return "xStep";
-        case 11:
-            return "Clutterfunk";
-        case 12:
-            return "Theory of Everything";
-        case 13:
-            return "Electroman Adventures";
-        case 14:
-            return "Clubstep";
-        case 15:
-            return "Electrodynamix";
-        case 16:
-            return "Hexagon Force";
-        case 17:
-            return "Blast Processing";
-        case 18:
-            return "Theory of Everything 2";
-        case 19:
-            return "Geometrical Dominator";
-        case 20:
-            return "Deadlocked";
-        case 21:
-            return "Fingerdash";
-        case 22:
-            return "Explorers";
-        case 23:
-            return "Firebird";
-        case 24:
-            return "The Seven Seas";
-        case 25:
-            return "Viking Arena";
-        case 26:
-            return "Airborne Robots";
-        case 27:
-            return "Secret";
-        case 28:
-            return "Payload";
-        case 29:
-            return "Beast Mode";
-        case 30:
-            return "Machina";
-        case 31:
-            return "Years";
-        case 32:
-            return "Frontlines";
-        case 33:
-            return "Space Pirates";
-        case 34:
-            return "Striker";
-        case 35:
-            return "Embers";
-        case 36:
-            return "Round 1";
-        case 37:
-            return "Monster Dance Off";
-        case 38:
-            return "Press Start";
-        case 39:
-            return "Nock Em";
-        case 40:
-            return "Power Trip";
-        default:
-            return "Stereo Madness";
-    }
+  return songIDs[songID] || songIDs[0];
 }
-
-module.exports = init;
\ No newline at end of file
+module.exports = init;

```

Comment:
not sure what got changed here?

Reply:
Not sure either. Don't know why it marked it as changed.